src/vsr/replica.zig

```diff
@@ -3318,21 +3318,14 @@ pub fn ReplicaType(
                 message.header.command == .do_view_change or
                 message.header.command == .start_view);
             assert(message.header.view > 0); // The initial view is already zero.
+            assert(self.status != .recovering); // Single node clusters don't have view changes.
```

Comment:
Would this be true also for non-single node clusters?

Reply:
Yes, because multi-node clusters always transition to a non-`recovering` status before `Replica.open()` finishes.